src/lib.rs

```diff
@@ -130,9 +132,13 @@ pub mod shared;
 #[macro_use]
 pub mod object;
 
+pub use auto::*;
+mod auto;
+
 mod bytes;
 pub mod error;
 mod file_error;
+pub mod key_file;
```

Comment:
We keep the modules private unless they provide some extensions (implementation details) needed by other `gtk-rs` crates not reexported in `lib.rs`.